Source/Core/Core/HW/WiimoteEmu/EmuSubroutines.cpp

```diff
@@ -53,7 +53,7 @@ void Wiimote::InvokeHandler(H&& handler, const WiimoteCommon::OutputReportGeneri
     return;
   }
 
-  (this->*handler)(Common::BitCastPtr<T>(rpt.data));
+  (this->*handler)(Common::BitCastPtr<T>(&rpt.data[0]));
```

Comment:
why this?

Reply:
personally I find it more clear, just a preference I guess.